app/models/loan.rb

```diff
@@ -211,4 +212,12 @@ def healthy?
   def health_status_available?
     return !health_check.nil?
   end
+
+  def agents_the_same?
+    (primary_agent.present? || secondary_agent.present?) && primary_agent == secondary_agent
+  end
+
+  def check_agents
+    errors.add(:primary_agent, 'for this project cannot be the same as the second point person') if agents_the_same?
```

Comment:
Thanks. Please move the error message into translations so that this can also be translated into Spanish. Let me know if you need help with this or if this is not clear. Thank you so much!

Reply:
sorry! forgot about that since a portion of it is translated
will do so now

Reply:
Thank you!